pages/linux/xeyes.md

```diff
@@ -0,0 +1,11 @@
+# xeyes
+
+> Display eyes on the screen that follow the mouse cursor.
+
+- Launch xeyes on local display:
+
+`xeyes`
+
+- Launch xeyes on a remote display:
+
+`xeyes -display {{remote_host}}:{{display}}.{{screen}}`
```

Comment:
I was leaning towards keeping the 0 values in the tokens and mention it in the description that those are values for the display and screen. That way, users can quickly run a basic form of the command, rather than looking up what might be the values for "display" and "screen".